src/osd/PGBackend.h

```diff
@@ -260,7 +260,8 @@ typedef std::shared_ptr<const OSDMap> OSDMapRef;
        const pg_stat_t &stat) = 0;
 
      virtual void schedule_recovery_work(
-       GenContext<ThreadPool::TPHandle&> *c) = 0;
+       GenContext<ThreadPool::TPHandle&> *c,
+       int cost) = 0;
```

Comment:
See branch, need to use uint64_t everywhere for cost.

Reply:
Sure, I used int since OpSchedulerItem takes in cost as int.